### PR TITLE
Instructions for STAC examples on own computer

### PR DESCRIPTION
**Pull request by samumantha**
* R 
* Pyhon

### CODE REVIEW

**Comment by samumantha** [comment]
Conda environment file could also be used directly from github, which could be updated after merge (link to raw file needed).

Could of course also still elaborate on running stuff etc, but no time now. Better than nothing for now. Can still update later :)

File: R/STAC/Readme.md [hunk 1]
```diff
@@ -27,3 +27,8 @@ In CSC Puhti supercomputer, the examples can be run with [r-env module](https://
 	* Time: 1:00:00 (or adjust to reasonable)
 * Click launch and wait until granted resources 
 * Click "Connect to RStudio Server" 
+
+
+If you want to run the example on your own computer, you'll need RStudio installed, along witht the following packages: ```geojsonsf, sf, terra, tidyverse, rstac, httr, gdalcubes```. Note that depending on your own setup you might also need to install further base packages. You can check out the R/STAC/install_stac.sh for possibly needed further installations, which is based on the [rocker/geospatial container and adjusted for this example](https://hub.docker.com/r/rocker/geospatial).
```
Comment (ktiits) [inline R/STAC/Readme.md:32]:
```suggestion
If you want to run the example on your own computer, you'll need RStudio installed, along with the following packages: ```geojsonsf, sf, terra, tidyverse, rstac, httr, gdalcubes```. 
```

File: R/STAC/Readme.md [hunk 2]
```diff
@@ -27,3 +27,8 @@ In CSC Puhti supercomputer, the examples can be run with [r-env module](https://
...
+
+Download the example code to your computer (either by copying the whole repository to your computer with git (`git clone https://github.com/csc-training/geocomputing.git`) or by downloading only the needed files via github webinterface (find R/STAC/STAC_CSC_example.Rmd` and in the upper right corner of the file "Download raw file button").
```
Comment (ktiits) [inline R/STAC/Readme.md:34]:
```suggestion
Download the example code to your computer (either by copying the whole repository to your computer with git (`git clone https://github.com/csc-training/geocomputing.git`) or by downloading only the needed files via github webinterface (find `R/STAC/STAC_CSC_example.Rmd` and in the upper right corner of the file "Download raw file button"). 
```

File: python/STAC/environment.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+name: STAC-workshop
```
Comment (ktiits) [inline python/STAC/environment.yml:1]:
```suggestion
name: stac
```

**Pull request merged**